hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightEndpoint.java

```diff
@@ -635,17 +636,21 @@ private void getPendingForDeletionDirInfo(
    * @return total used data size in bytes
    * @throws IOException ioEx
    */
-  protected long fetchSizeForDeletedDirectory(long objectId)
+  protected Pair<Long, Long> fetchSizeForDeletedDirectory(long objectId)
```

Comment:
It would be good to update the javadoc for this method.
Currently it says return total data size (no replication)
It should be changed to:
```
/**
   * Given an object ID, return total data size as a pair of Total Size, Total Replicated Size
   * under this object. Note:- This method is RECURSIVE.
   *
   * @param objectId the object's ID
   * @return total used data size and replicated total used data size in bytes
   * @throws IOException ioEx
   */
```